Source/Scene/Model.js

```diff
@@ -332,7 +333,7 @@ define([
      * @param {String} [options.basePath=''] The base path that paths in the glTF JSON are relative to.
      * @param {Boolean} [options.show=true] Determines if the model primitive will be shown.
      * @param {Matrix4} [options.modelMatrix=Matrix4.IDENTITY] The 4x4 transformation matrix that transforms the model from model to world coordinates.
-     * @param {Number} [options.scale=1.0] A uniform scale applied to this model.
+     * @param {Number|Cartesian3} [options.scale=new Cartesian3(1.0, 1.0, 1.0)] A scale applied to this model.
```

Comment:
Can we avoid overloading this property in the low-level code?  Maybe deprecate and replace with `nonUniformScale` or see if this can be implemented with `modelMatrix`?  @lilleyse may have a better suggestion.